include/roboteam_ai/bt/Node.hpp

```diff
@@ -4,8 +4,22 @@
 #include <vector>
 #include <cstdio>
 #include <cstdarg>
+#include <include/roboteam_ai/world/World.h>
```

Comment:
This include shouldn't be necessary right? Since every source file can use it if it has the include for world already.